Sources/MongoSwift/Client.swift

```diff
@@ -1,13 +1,23 @@
 import Foundation
 import libmongoc
 
-public struct ClientOptions {
+public struct ClientOptions: BsonEncodable {
```

Comment:
we don't actually send these options now anyway but I added the `encode` method so that we don't forget `eventMonitoring` should be omitted